Sweet Module example is not working

http://www.sweetjs.org/doc/1.0/tutorial.html#_sweet_modules

I tried the example given at the above link and it is giving me below error

/usr/lib/node_modules/@sweet-js/cli/node_modules/resolve/lib/sync.js:42
    throw err;
    ^

Error: Cannot find module './es2015-macros'

Updated the path in import statement to 'import { class } from './es2015-macros.js'; ' i.e. added .js in the module name. Re-run the example but getting below error this time

/usr/lib/node_modules/@sweet-js/cli/node_modules/sweet-core/dist/syntax.js:360
          throw new Error('Scopeset ' + debugBase + ' has ambiguous subsets ' + debugAmbigousScopesets);
          ^

Error: Scopeset {outsideEdge_5} has ambiguous subsets {outsideEdge_5}, {outsideEdge_5}
    at Syntax.resolve (/usr/lib/node_modules/@sweet-js/cli/node_modules/sweet-core/dist/syntax.js:360:17)

--- a/src/node-loader.js
+++ b/src/node-loader.js
@@ -14,7 +14,7 @@ export default class NodeLoader extends SweetLoader {
 
   constructor(baseDir: string, options: NodeLoaderOptions = {}) {
     super(baseDir, options);
-    this.extensions = options.extensions || [];
+    this.extensions = options.extensions || ['.js', '.mjs'];
   }
 
   normalize(name: string, refererName?: string, refererAddress?: string) {
@@ -23,7 +23,7 @@ export default class NodeLoader extends SweetLoader {
     if (match && match.length >= 3) {
       let resolvedName = resolve.sync(match[1], {
         basedir: refererName ? dirname(refererName) : this.baseDir,
-        extensions: this.extensions ? this.extensions : ['.js'],
+        extensions: this.extensions,
       });
       return `${resolvedName}:${match[2]}`;
     }

--- a/test/unit/test-node-loader.js
+++ b/test/unit/test-node-loader.js
@@ -1,0 +1,7 @@
+import test from 'ava';
+import NodeLoader from '../../src/node-loader';
+
+test('a node loader should normalize a path that is missing the extension', t => {
+  let loader = new NodeLoader(__dirname);
+  t.regex(loader.normalize('./test-node-loader'), /.*test-node-loader\.js/);
+});

